build/webpack.base.config.js

```diff
@@ -149,6 +154,14 @@ module.exports = {
                                 loader: 'svg-inline-loader',
                                 options: config.svgInlineLoaderOptions,
                             },
+                        }, {
+                            resourceQuery: /http/,
+                            use: {
+                                loader: 'file-loader',
+                                options: {
+                                    name: fileLoaderName,
+                                },
+                            },
```

Comment:
Allow importing `file.svg?http` to force the file to generate an `<img src="...">` that fetches the file over HTTP.  Otherwise, SVG's under 8k will be base64/utf8 encoded and inlined into the JS bundle which may bloat the bundle size if the images are not guaranteed to be used.